AltStore/Views/Settings/DevModeView.swift

```diff
@@ -0,0 +1,190 @@
+                            isShowingIncorrectPasswordAlert = false
+                            isShowingPasswordAlert = true
+                        })
+                        SwiftUI.Button("Cancel", action: {
+                            isShowingIncorrectPasswordAlert = false
+                            isShowingDevModePrompt = false
+                        })
+                    }
+            } else {
+                view
+            }
+        }
+    }
+    
+    func enableDevMode() {
+        UserDefaults.standard.isDevModeEnabled = true
+        isShowingDevModePrompt = false
+        isShowingDevModeMenu = true
+    }
+    
+    func tickCountdown() {
+        if countdown <= 0 { return }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+            countdown -= 1
+            tickCountdown()
+        }
+    }
+}
+
+struct DevModeMenu: View {
+    @ObservedObject private var iO = Inject.observer
+    
+    @AppStorage("isConsoleEnabled")
+    var isConsoleEnabled: Bool = false
+    
+    var body: some View {
+        List {
+            Section {
+                Toggle(L10n.DevModeView.console, isOn: self.$isConsoleEnabled)
+                    .onChange(of: self.isConsoleEnabled) { value in
+                        LCManager.shared.isVisible = value
+                    }
+                
+                NavigationLink(L10n.DevModeView.dataExplorer) {
+                    FileExplorer(url: FileManager.default.altstoreSharedDirectory)
+                        .navigationTitle(L10n.DevModeView.dataExplorer)
+                }.foregroundColor(.red)
+                
+                NavigationLink(L10n.DevModeView.tmpExplorer) {
+                    FileExplorer(url: FileManager.default.temporaryDirectory)
+                        .navigationTitle(L10n.DevModeView.tmpExplorer)
+                }.foregroundColor(.red)
+                
+                Toggle(L10n.DevModeView.skipResign, isOn: ResignAppOperation.skipResignBinding)
+                    .foregroundColor(.red)
+            } footer: {
+                Text(L10n.DevModeView.skipResignInfo)
+            }
+            
+            Section {
+                NavigationLink(L10n.DevModeView.Minimuxer.stagingExplorer + " (Coming soon, needs minimuxer additions)") {
+                    FileExplorer(url: FileManager.default.altstoreSharedDirectory)
+                        .navigationTitle(L10n.DevModeView.Minimuxer.stagingExplorer)
+                }.foregroundColor(.red).disabled(true)
+                
+                NavigationLink(L10n.DevModeView.Minimuxer.viewProfiles + " (Coming soon, needs minimuxer additions)") {
+                    
+                }.disabled(true)
+                
+                SwiftUI.Button(L10n.DevModeView.Minimuxer.dumpProfiles + " (Coming soon, needs minimuxer additions)", action: {
+                    // TODO: dump profiles to Documents/ProfileDump/[current time]
```

Comment:
Are the changes already implemented in minimuxer? If so, it would be great if you could resolve the TODO here. Otherwise this will just have to wait and I'll merge these changes including the TODO first.

Reply:
They aren't, I'm currently working on refactoring minimuxer to allow for better usage from Swift. After that, they should be pretty easy to implement